ev3dev/core.py

```diff
@@ -61,6 +61,9 @@
 INPUT_AUTO = ''
 OUTPUT_AUTO = ''
 
+# Keep the __version__ in sync with the package version in debian/changelog
+__version___ = '0.7.0'
```

Comment:
triple underscore...good eye that is a typo, will fix.

There is probably a way to do this automatically but we aren't doing releases at a frantic pace so for now I think this is fine.  We can automate it down the road.